sklearn/utils/_metadata_requests.py

```diff
@@ -1051,6 +1051,10 @@ class RequestMethod:
         function, e.g. ``["sample_weight"]`` if the corresponding method
         accepts it as a metadata.
 
+    validate_keys : bool, default=True
+        Whether to check if the requested parameters fit the actual parameters
+        of the method.
```

Comment:
note: this was added so that we could simply add an instance of this descriptor to `CheckingClassifier`